.github/workflows/run_relup_tests.yaml

```
@@ -14,101 +14,117 @@ on:
+      CUR_EE_VSN: "${{ needs.relup_test_plan.outputs.CUR_EE_VSN }}"
+    defaults:
+      run:
+        shell: bash
+    steps:
+      # setup Erlang to run lux
+    - uses: erlef/setup-beam@v1
       with:
-        repository: emqx/emqtt-bench
-        ref: 0.3.4
-        path: emqtt-bench
+        otp-version: "24.2"
     - uses: actions/checkout@v2
       with:
         repository: hawk/lux
-        ref: lux-2.6
+        ref: lux-2.8.1
         path: lux
-    - uses: actions/checkout@v2
-      with:
-        repository: ${{ github.repository }}
-        path: emqx
-        fetch-depth: 0
-
-    - name: Get old vsn
-      run: echo "OLD_VSNS=$(emqx/scripts/relup-base-vsns.sh ${{ matrix.profile }} | xargs echo -n)" >> $GITHUB_ENV
-      run: echo "VSN=$(emqx/pkg-vsn.sh ${{ matrix.profile }})" >> $GITHUB_ENV
-
-    - name: build emqx
-      env:
-        PROFILE: ${{ matrix.profile }}
-      run: make -C emqx ${PROFILE}-tgz
-    - name: build emqtt-bench
-      run: make -C emqtt-bench
-    - name: build lux
+    - name: Install lux
       run: |
         set -e -u -x
         cd lux
         autoconf
         ./configure
         make
-        make install
+        echo "$(pwd)/bin" >> $GITHUB_PATH
+    - uses: actions/download-artifact@v2
+      name: Download built emqx and test scenario
+      with:
+        name: emqx_built
+        path: .
     - name: run relup test
-      env:
-        PROFILE: ${{ matrix.profile }}
-      timeout-minutes: 20
       run: |
         set -e -x -u
-        if [ -n "$OLD_VSNS" ]; then
-            mkdir -p packages
-            cp emqx/_packages/${PROFILE}/*.tar.gz packages
-            cp emqx/_upgrade_base/*.tar.gz packages
-            lux \
-            --case_timeout infinity \
-            --var PROFILE=$PROFILE \
-            --var PACKAGE_PATH=$(pwd)/packages \
-            --var BENCH_PATH=$(pwd)/emqtt-bench \
-            --var ONE_MORE_EMQX_PATH=$(pwd)/one_more_emqx \
-            --var VSN="$VSN" \
-            --var OLD_VSNS="$OLD_VSNS" \
-            emqx/.ci/fvt_tests/relup.lux
+        chmod a+x scripts/**/*.sh
+        ls -l scripts
+        ls -l scripts/relup-test
+        case "$OLD_VSN" in
+          e*)
+            export CUR_VSN="$CUR_EE_VSN"
+            ;;
+          v*)
+            export CUR_VSN="$CUR_CE_VSN"
+            ;;
+          *)
+            echo "unknown old version $OLD_VSN"
+            exit 1
+            ;;
+        esac
+        mkdir -p lux_logs
+        if ! ./scripts/relup-test/run-relup-lux.sh $OLD_VSN; then
+          docker logs node1.emqx.io | tee lux_logs/emqx1.log
+          docker logs node2.emqx.io | tee lux_logs/emqx2.log
+          exit 1
         fi
-    - uses: actions/upload-artifact@v1
+    - uses: actions/upload-artifact@v2
+      name: Save debug data
       if: failure()
       with:
-        name: lux_logs
-        path: lux_logs
+        name: debug_data
+        path: |
+          wd/lux_logs
```

I didn't see a `wd` folder above.  Maybe just `lux_logs`?

```suggestion
          lux_logs
```